### ISSUE TITLE
Joint measurability of heat kernel

### ISSUE THREAD

**Issue by pabryan** [pabryan/rough_metrics_heat_kernel_regularity#12]
For point (vi) in proposition 4.2, do we get joint measurability in \(x\) and \(y\)? This is not true in general. See for example:

[](https://math.stackexchange.com/questions/647235/counterexample-to-measurable-in-each-variable-separately-implies-measurable)

**Comment by lashputin** [comment]
Yep. I think we should drop joint measurability. We only need separately measurable.
